worlds/doom_1993/Items.py

```diff
@@ -0,0 +1,853 @@
+    350106: {'classification': ItemClassification.progression,
+             'count': 1,
+             'progression_count': 0,
+             'name': 'Backpack',
+             'doom_type': 8,
+             'episode': -1,
+             'map': -1},
+    350107: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Armor',
+             'doom_type': 2018,
+             'episode': -1,
+             'map': -1},
+    350108: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Mega Armor',
+             'doom_type': 2019,
+             'episode': -1,
+             'map': -1},
+    350109: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Berserk',
+             'doom_type': 2023,
+             'episode': -1,
+             'map': -1},
+    350110: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Invulnerability',
+             'doom_type': 2022,
+             'episode': -1,
+             'map': -1},
+    350111: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Partial invisibility',
+             'doom_type': 2024,
+             'episode': -1,
+             'map': -1},
+    350112: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Supercharge',
+             'doom_type': 2013,
+             'episode': -1,
+             'map': -1},
+    350113: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Medikit',
+             'doom_type': 2012,
+             'episode': -1,
+             'map': -1},
+    350114: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Box of bullets',
+             'doom_type': 2048,
+             'episode': -1,
+             'map': -1},
+    350115: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Box of rockets',
+             'doom_type': 2046,
+             'episode': -1,
+             'map': -1},
+    350116: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Box of shotgun shells',
+             'doom_type': 2049,
+             'episode': -1,
+             'map': -1},
+    350117: {'classification': ItemClassification.filler,
+             'count': 0,
+             'progression_count': 0,
+             'name': 'Energy cell pack',
+             'doom_type': 17,
+             'episode': -1,
+             'map': -1},
+}
+
+
```

Comment:
To avoid issues down the line (in case you change the name of an item, add a new one, etc.), I recommend adding a "group" property to the class, where you can assign these item_name_groups there, then just write a little bit of code down here to add them to their groups. Though, I guess if these are auto-generated, then it's probably fine.

Reply:
Generated code